### PR TITLE
Calculator:Milestone-2

### PR DESCRIPTION
**Pull request by jebitok-dev**
In this milestone I:-
- created the components Display, ButtonPanel, with buttons then linked them to the main App component.
- made a basic layout of the calculator app
- ensured the app live version is up

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @jebitok-dev! So far so good! :+1: There just a few changes for you to make to complete this milestone. These include:
- [ ] Place the `App` component in the `src/components` directory ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png))
- [ ] `App`'s children components should be wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))
- [ ] Your `ButtonPanel` component should include __all__ the following buttons divided in the groups below: ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png))
  * Group 1: AC, +/-, %, ÷
  * Group 2: 7, 8, 9, X
  * Group 3: 4, 5, 6, -
  * Group 4: 1, 2, 3, +
  * Group 5: 0, ., =

__Please make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__[Treasure Kabareebe](https://trekab.github.io)__

File: src/components/buttonPanel.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            {this.renderButton('AC')}
+            {this.renderButton('+/-')}
+            {this.renderButton('%')}
+            {this.renderButton('/')}
+          </div>
+          <div className="calculator-row">
+            {this.renderButton('7')}
+            {this.renderButton('8')}
+            {this.renderButton('9')}
+            {this.renderButton('÷')}
```
Comment (afanimartin) [inline src/components/buttonPanel.js]: - This group is lacking the `X` button. This was already pointed out. Carefully review comments by TSEs.

**Review by afanimartin — CHANGES_REQUESTED** [review]
### Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work making the requested changes.
Let's however make one more change before merging.
- Add the `X` button to the `ButtonDisplay` component.

### Martin Afani
#### Slack: whiz25

**Review by afanimartin — APPROVED** [review]
## Approved 👏🏿 👏🏿 👏🏿 
Nice work making the requested changes. Merge and proceed to the next milestone.

### Martin Afani
#### Slack: whiz25

**Pull request merged**